modules/data-loading/partials/create-connector.adoc

```diff
@@ -0,0 +1,10 @@
+=== Create connector
+Once you have prepared the configuration file, run command `gadmin connector create` and specify the configuration file to create the connector:
+
+  gadmin connector create --c <config_file>
+
+The connectors start streaming from the data source immediately after creation if the configurations are valid.
+You can run `gadmin connector status` to verify the status of the connectors.
+If the configurations are valid, the connectors should be in `RUNNING` status.
+
+Data streamed from the source stays in TigerGraph's internal Kafka topics until they are ingested by a loading job.
```

Comment:
The target kafka topic does not have to be tigergraph kafka, it can be any accessible kafka

Reply:
What would be the use case for users to stream to another Kafka cluster in this case? Is that something we want to cover?

Reply:
Yes. Kafka loader can load data from external Kafka. 
A use case would be tigergraph kafka cluster does not have enough disk but have large memory and CPU, and they have another kafka cluster with huge disk to just store data. 
Double check, do we want to cover this case? cc @chengjie-qin